Canvas git integration limitations (#8399)
fishtown-analytics/docs.getdbt.com: website/snippets/_canvas-prerequisites.md

    - Snowflake
    - Trino
    - You can access the <Constant name="visual_editor" /> with adapters not listed, but some features may be missing at this time. 
- Use [GitHub](/docs/cloud/git/connect-github), [GitLab](/docs/cloud/git/connect-gitlab), or [Azure DevOps](/docs/cloud/git/connect-azure-devops) as your <Constant name="git" /> provider, connected to dbt via HTTPS. SSH connections are not supported for <Constant name="visual_editor" /> at this time.

>   - Note that <Constant name="visual_editor" /> doesn't support Git integration for on-premises or [single tenant deployments](/docs/cloud/about-cloud/tenancy) at this time.

Author: ```suggestion
  - <Constant name="visual_editor" /> doesn't support Git integration for GitHub self-hosted / on-premises deployments.
```

    - Trino
    - You can access the <Constant name="visual_editor" /> with adapters not listed, but some features may be missing at this time. 

> - Be using [GitHub](/docs/cloud/git/connect-github), [GitLab](/docs/cloud/git/connect-gitlab), or [Azure DevOps](/docs/cloud/git/connect-azure-devops) as your <Constant name="git" /> provider, connected to dbt via HTTPS. SSH connections are not supported for <Constant name="visual_editor" /> at this time.

Author: ```suggestion
- Use [GitHub](/docs/cloud/git/connect-github), [GitLab](/docs/cloud/git/connect-gitlab), or [Azure DevOps](/docs/cloud/git/connect-azure-devops) as your <Constant name="git" /> provider, connected to dbt via HTTPS. SSH connections are not supported for <Constant name="visual_editor" /> at this time.
```

>   - **Note:** <Constant name="git" /> integration is not available for <Constant name="visual_editor" /> on on-premises or single tenant deployments at this time.

Author: ```suggestion
  - Note that <Constant name="visual_editor" /> doesn't support Git integration for on-premises or [single tenant deployments](/docs/cloud/about-cloud/tenancy) at this time.
```
